pkg/config/system_mask.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package config
+
+import (
+	"sort"
+
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+)
+
+// SystemConfigMask is a mask that can be applied to a set of system config
+// entries to filter out unwanted entries.
+type SystemConfigMask struct {
+	keys []roachpb.Key
```

Comment:
nit: name this `allowed` so that it's clear this is an allowlist?

Reply:
Done.